webapp/src/dogDetails/dogDetails.tsx

```diff
@@ -312,10 +313,10 @@ const DogDetails = (props: any) => {
                 displayEmpty
                 renderValue={(selected: any) => (
                   <div className={classes.chips}>
-                    {(selected as string[]).map((value) => (
+                    {(selected as IBehaviour[]).map((value) => (
```

Comment:
Probably revert it since group C will change their behaviours return to list of strings